tests/unittests/CoreText/CTFontTests.mm

```diff
@@ -513,4 +515,203 @@ virtual void TearDown() {
+                                                          { kCGPathElementMoveToPoint, { 421, 697 } },
+                                                          { c_CGPathElementCurveType, { 454, 696.5 } },
+                                                          { c_CGPathElementCurveType, { 477, 696 } },
+                                                          { c_CGPathElementCurveType, { 769.5, 780 } },
+                                                          { c_CGPathElementCurveType, { 868, 994 } },
+                                                          { c_CGPathElementCurveType, { 788.5, 1200.5 } },
+                                                          { c_CGPathElementCurveType, { 578, 1280 } },
+                                                          { c_CGPathElementCurveType, { 421, 1261 } },
+                                                          { kCGPathElementCloseSubpath, {} } });
+
+    // Struct of external variables to pass to CGPathApplyFunction (which does not support lambdacaptures)
+    struct ComparePathContext {
+        size_t count;
+        std::vector<ExpectedCGPathElement> expectedElements;
+    };
+
+    ComparePathContext comparePathContext = { 0, expectedElements };
+
+    CGPathApplierFunction comparePathToExpected = [](void* context, const CGPathElement* element) {
+        ComparePathContext* comparePathContext = reinterpret_cast<ComparePathContext*>(context);
+        EXPECT_EQ(comparePathContext->expectedElements[comparePathContext->count].type, element->type);
+
+        // Error margin is slightly higher for this test
+        switch (element->type) {
+            case kCGPathElementMoveToPoint:
+            case kCGPathElementAddLineToPoint:
+                EXPECT_NEAR(comparePathContext->expectedElements[comparePathContext->count].point.x, element->points[0].x, 0.002f);
+                EXPECT_NEAR(comparePathContext->expectedElements[comparePathContext->count].point.y, element->points[0].y, 0.002f);
+                break;
+
+            case kCGPathElementAddQuadCurveToPoint:
+                EXPECT_NEAR(comparePathContext->expectedElements[comparePathContext->count].point.x, element->points[1].x, 0.002f);
+                EXPECT_NEAR(comparePathContext->expectedElements[comparePathContext->count].point.y, element->points[1].y, 0.002f);
+                break;
+
+            case kCGPathElementAddCurveToPoint:
+                EXPECT_NEAR(comparePathContext->expectedElements[comparePathContext->count].point.x, element->points[2].x, 0.002f);
+                EXPECT_NEAR(comparePathContext->expectedElements[comparePathContext->count].point.y, element->points[2].y, 0.002f);
+                break;
+
+            case kCGPathElementCloseSubpath:
+                // No-op, CloseSubpath has no associated point
+                break;
+
+            default:
+                EXPECT_TRUE_MSG(false, @"An invalid CGPathElementType was returned from CTFontCreatePathForGlyph");
```

Comment:
Or, `ADD_FAILURE_AT() << "msg"` to make it a non-fatal failure.

---
In reply to: [86888788](https://github.com/Microsoft/WinObjC/pull/1330#discussion_r86888788) [](ancestors = 86888788)